.madrun.js

```diff
@@ -57,12 +61,22 @@ module.exports = {
     'wisdom:type': () => 'bin/release.js',
     'docker:pull:node': () => 'docker pull node',
     'docker:pull:alpine': () => 'docker pull mhart/alpine-node',
-    'docker:push': () => `docker push coderaiser/cloudcmd:${version}`,
-    'docker:push:latest': () => 'docker push coderaiser/cloudcmd:latest',
+    'docker:pull:arm32': () => 'docker pull arm32v7/node:slim',
+    'docker:pull:arm64': () => 'docker pull arm64v8/node:slim',
+    'docker:push': () => `docker push coderaiser/cloudcmd:${version}-x64`,
+    'docker:push:latest': () => 'docker push coderaiser/cloudcmd:latest-x64',
     'docker:push:alpine': () => `docker push coderaiser/cloudcmd:${version}-alpine`,
     'docker:push:alpine:latest': () => 'docker push coderaiser/cloudcmd:latest-alpine',
-    'docker:build': () => `docker build -t coderaiser/cloudcmd:${version} .`,
-    'docker:build:alpine': () => `docker build -f Dockerfile.alpine -t coderaiser/cloudcmd:${version}-alpine .`,
+    'docker:push:arm32': () => dockerPush('arm32', version),
+    'docker:push:arm32:latest': () => dockerPush('arm32'),
+    'docker:push:arm64': () => dockerPush('arm64', version),
+    'docker:push:arm64:latest': () => dockerPush('arm64'),
+    'docker:build': () => `docker build -f docker/Dockerfile -t coderaiser/cloudcmd:${version}-x64 .`,
+    'docker:build:alpine': () => `docker build -f docker/Dockerfile.alpine -t coderaiser/cloudcmd:${version}-alpine .`,
+    'docker:build:arm32': () => `docker build -f docker/arm/Dockerfile.arm32v7 -t coderaiser/cloudcmd:${version}-arm32 .`,
+    'docker:build:arm64': () => `docker build -f docker/arm/Dockerfile.arm64v8 -t coderaiser/cloudcmd:${version}-arm64 .`,
+    'docker:manifest:create': () => 'docker manifest create coderaiser/cloudcmd:latest coderaiser/cloudcmd:latest-x64 coderaiser/cloudcmd:latest-arm32 coderaiser/cloudcmd:latest-arm64',
```

Comment:
What do you think about creating a function for `docker build`, `docker tag`, `docker rmi` to simplify things a bit

Reply:
done ✅